examples/nlp/language_modeling/conf/megatron_ul2_config.yaml

```diff
@@ -91,22 +69,22 @@ model:
     merge_file: null
     num_sentinel_tokens: 100
 
+  # weight init
+  embedding_init_method_std: 0.02 # Standard deviation of the zero mean normal distribution used for weight initialization.')
```

Comment:
It looks like `embedding_init_method_std` and `embedding_dropout` use the same values in multiple models. Should they be moved to base config?

Reply:
My thinking was that the base model should only have `ParallelTransformer` related args and nothing else. And so embedding-related args are outside of this. I believe I've handled backward compact for older models by setting this to be equal to `model.init_method_std`. Same for embedding dropout etc.